Polyfills/Window/CMakeLists.txt

```diff
@@ -3,16 +3,31 @@ set(SOURCES
     "Source/Window.h"
     "Source/Window.cpp")
 
+if(WIN32)
+    set(SOURCES ${SOURCES} "Source/Windows/WindowPlatform.h")
+endif()
+
 add_library(Window ${SOURCES})
 warnings_as_errors(Window)
 
 target_include_directories(Window
     PUBLIC "Include")
 
+if(WIN32)
+    target_include_directories(Window 
+        PRIVATE "Source/Windows")
+endif()
+
 target_link_to_dependencies(Window 
     PUBLIC napi
     PRIVATE base-n
     PRIVATE JsRuntime)
 
+if(WIN32)
```

Comment:
>WIN32 [](start = 3, length = 5)

iirc win32 evaluates to defined for cmake generated uwp solutions. I am not familiar with the below lib that is introduced by this change, but if it causes us to consume the windows runtime library it will break BabylonReactNative compilation. Do we know if shlwapi.lib is uwp friendly?

Reply:
I have no idea, I never worked with UWP prior to this. @syntheticmagus @bghgary do you have any idea?

Reply:
CMake defines WIN32 for all windows platforms. WINDOWS_STORE is additionally defined for UWP. `shlwapi` is desktop only. There are hacks to make it work in UWP, but it probably won't pass [WACK](https://developer.microsoft.com/en-us/windows/downloads/app-certification-kit/).

Reply:
We can switch off of `BABYLON_NATIVE_PLATFORM` thankfully somebody has already had the foresight to make sure that we aren't overlapping our build configuration for Win32 and UWP:

https://github.com/BabylonJS/BabylonNative/blob/b4f81771abb2722c3c18731a59b31c18298fcd40/CMakeLists.txt#L17